burn-ndarray/src/backend.rs

```diff
@@ -20,7 +28,7 @@ impl Default for NdArrayDevice {
 
 #[derive(Clone, Copy, Default, Debug)]
 pub struct NdArrayBackend<E> {
-    _e: E,
+    phantom: PhantomData<E>,
```

Comment:
What is `PhantomData`  does it remove the unecesasry element contained in the struct? If yes that's awesome.

Reply:
That's exactly what it does. Plus it's zero sized type. It has the effect of _e but clearer and without memory over head.

Reply:
https://doc.rust-lang.org/std/marker/struct.PhantomData.html